pom.xml

```diff
@@ -560,7 +560,7 @@
             <dependency>
                 <groupId>org.apache.solr</groupId>
                 <artifactId>solr-solrj</artifactId>
-                <version>9.0.0</version>
+                <version>9.4.0</version>
```

Comment:
Hmm ved vi at den virker? Er den den DE bruger?

Reply:
Ifølge solr4j dokumentation virker det, men jeg kan ikke lige huske hvorfor det var at jeg havde brug for at opgradere den... Jeg spørg dem i morgen om de har nogen dårlige erfaringer med 9.4.0